capacity4more/modules/c4m/admin/c4m_admin_groups/c4m_admin_groups.module

```diff
@@ -5,3 +5,11 @@
  */
 
 include_once 'c4m_admin_groups.features.inc';
+
+/**
+ * Implements hook_init().
+ */
+function c4m_admin_groups_init() {
+  $settings['machineName'] = array();
```

Comment:
this is added on every page -- can we do it just for the pages needed -- preferebly via hook_form_alter()